domains/nonamepro.is-cool.dev.json

```diff
@@ -0,0 +1,16 @@
+{
+  "description": "Personal Self Hosted Apps and Public Website",
+
+  "domain": "is-cool.dev",
+  "subdomain": "nonamepro",
+
+  "owner": {
+    "email": "noname299199@gmail"
+  },
+
+  "record": {
+    "NS": ["ns1.infinityfree.com","ns2.infinityfree.com"]

```

Comment:
We no longer support NS records as per the readme.md, please remove them and use alternative records.